@xen-orchestra/web/src/components/host/HostPifsTable.vue

```diff
@@ -0,0 +1,257 @@
+      <template #actions>
+        <UiButton
+          v-tooltip="$t('coming-soon')"
+          disabled
+          size="medium"
+          variant="secondary"
+          accent="info"
+          :left-icon="faArrowsRotate"
+        >
+          {{ $t('scan-pifs') }}
+        </UiButton>
+      </template>
+    </UiTitle>
+    <div class="container">
+      <div class="table-actions">
+        <UiQuerySearchBar @search="(value: string) => (searchQuery = value)" />
+        <UiTableActions>
+          <UiButton
+            v-tooltip="$t('coming-soon')"
+            disabled
+            :left-icon="faEdit"
+            variant="tertiary"
+            accent="info"
+            size="medium"
+          >
+            {{ $t('edit') }}
+          </UiButton>
+          <UiButton
+            v-tooltip="$t('coming-soon')"
+            disabled
+            :left-icon="faTrash"
+            variant="tertiary"
+            accent="danger"
+            size="medium"
+          >
+            {{ $t('delete') }}
+          </UiButton>
+          <template #title>
+            <UiActionsTitle> {{ $t('table-actions') }}</UiActionsTitle>
+          </template>
+        </UiTableActions>
+
+        <UiTopBottomTable :selected-items="0" :total-items="0" />
+      </div>
+      <VtsDataTable :is-ready :has-error :no-data-message="pifs.length === 0 ? $t('no-network-detected') : undefined">
+        <template #thead>
+          <tr>
+            <template v-for="column of visibleColumns" :key="column.id">
+              <th v-if="column.id === 'checkbox'" v-tooltip="$t('coming-soon')" class="checkbox">
+                <UiCheckbox :v-model="areAllSelected" accent="info" disabled />
+              </th>
+              <th v-else-if="column.id === 'more'" class="more">
+                <UiButtonIcon v-tooltip="$t('coming-soon')" :icon="faEllipsis" accent="info" disabled size="small" />
+              </th>
+              <th v-else>
+                <div v-tooltip class="text-ellipsis">
+                  <VtsIcon accent="brand" :icon="headerIcon[column.id]" />
+                  {{ column.label }}
+                </div>
+              </th>
+            </template>
+          </tr>
+        </template>
+        <template #tbody>
+          <tr
+            v-for="row of rows"
+            :key="row.id"
+            :class="{ selected: selectedPifId === row.id }"
+            @click="selectedPifId = row.id"
+          >
+            <td
+              v-for="column of row.visibleColumns"
+              :key="column.id"
+              class="typo p2-regular"
+              :class="{ checkbox: column.id === 'checkbox' }"
+            >
+              <div v-if="column.id === 'checkbox'" v-tooltip="$t('coming-soon')">
+                <UiCheckbox v-model="selected" accent="info" :value="row.id" disabled />
+              </div>
+              <div v-else-if="column.id === 'status'" class="status">
+                <PifStatus :network="getNetwork(row.value)" />
```

Comment:
Not needed, you can use the `VtsConnectionStatus` directly, like on the Pool Network Table, and compute the status directly in this component.
There is also some logic to fix to compute the correct statuses, I will let you see with @J0ris-K for details.